quinn/src/platform/fallback.rs

```diff
@@ -2,20 +2,29 @@ use std::{io, net::SocketAddr};
 
 use mio::net::UdpSocket;
 
-use proto::EcnCodepoint;
+use proto::{EcnCodepoint, Transmit};
 
 impl super::UdpExt for UdpSocket {
     fn init_ext(&self) -> io::Result<()> {
         Ok(())
     }
 
-    fn send_ext(
-        &self,
-        remote: &SocketAddr,
-        _: Option<EcnCodepoint>,
-        msg: &[u8],
-    ) -> io::Result<usize> {
-        self.send_to(msg, remote)
+    fn send_ext(&self, transmits: &[Transmit]) -> io::Result<usize> {
+        let mut sent = 0;
+        for transmit in transmits {
+            match self.send_to(&transmit.contents, &transmit.destination) {
+                Ok(_) => {
+                    sent += 1;
+                }
+                Err(_) if sent != 0 => {
```

Comment:
What errors are being swallowed here? Maybe add a comment to clarify why this is okay?

Reply:
Done.